### PR TITLE
Add new `Resources` chapter

### PR DESCRIPTION
**Pull request by Bosch-0**
As part of our [design re-org ](https://github.com/BitcoinDesign/Guide/issues/725) efforts this PR introduces a new chapter called `Resources`. 

Closes https://github.com/BitcoinDesign/Guide/issues/720 

[Live preview](https://deploy-preview-729--sad-borg-390916.netlify.app/guide/resources)

### CODE REVIEW

File: guide/contribute/contribute.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ These contribution guidelines are for the Bitcoin Design Guide project. To get i
 ---
 
 {% include next-previous.html
-   previousUrl = "/guide/case-studies/shared-account/"
+   previousUrl = "/guide/resources/"
    previousName = "Shared account"
```
Comment (GBKS) [inline guide/contribute/contribute.md:78]:
```suggestion
   previousName = "Resources"
```

File: guide/contribute/formatting.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ There should be whitespace between paragraphs.
 ```markdown
 > The details are not the details. They make the design.
 >
-> <cite>By <a href="#">Charles Eames</a></cite>
+> This is a blockquote
```
Comment (GBKS) [inline guide/contribute/formatting.md:88]: Did you change this intentionally? Doesn't match the code right below anymore and is also not the proper formatting for the author name.

File: guide/resources/design-research.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
 ---
 layout: guide
-title: User research resources
-description: Surveys, academic papers, articles and other resources on bitcoin user research.
-nav_order: 1
-parent: User research
-grand_parent: Designing bitcoin products
-permalink: /guide/designing-products/user-research/resources
+title: Design research
```
Comment (GBKS) [inline guide/resources/design-research.md:3]: Interesting decision to switch from `User research` to `Design research`. Trying to figure out if I agree or not...

File: guide/resources/design-research.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
+description: Bitcoin design related journal articles, independent research, case studies, and blog posts & write-ups.
+nav_order: 2
+parent: Resources
+permalink: /guide/resources/design-research/
 main_classes: -no-top-padding
-image: https://bitcoin.design/assets/images/guide/designing-products/user-research-resources/user-research-resources-preview.jpg
+image: https://bitcoin.design/assets/images/guide/resources/design-research/user-research-resources-preview.jpg
+redirect_from:
+ - /guide/designing-products/user-research/resources
```
Comment (GBKS) [inline guide/resources/design-research.md:11]:
```suggestion
 - /guide/designing-products/user-research/resources/
```

File: guide/resources/introduction.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+layout: guide
+title: Resources
+description: Resources to help you build better bitcoin products.
+nav_order: 8
+has_children: true
+permalink: /guide/resources
```
Comment (GBKS) [inline guide/resources/introduction.md:7]:
```suggestion
permalink: /guide/resources/
```

Jekyll is a bit annoying with the `/` at the end of links. Specifically, rake test will see URLs with and without the last `/` as different and throw errors. So we need to always add it.

**Review by GBKS — CHANGES_REQUESTED** [review]
Looks great, just some minor comments.

Make sure to run `rake test` for this type of PR to catch broken URLs. It's throwing 5 errors (1 of which can be ignored).

We can also add a link to the moved `Design resources` page from the bottom of the `User research` page, right [here](https://deploy-preview-729--sad-borg-390916.netlify.app/guide/designing-products/user-research/#resources). Good to cross-link those two pages now that they are not next to each other in the nav anymore.

File: guide/resources/design-research.md (hunk 3 above)
Comment (Bosch-0) [reply guide/resources/design-research.md:3]: Made more sense to me, some of the research contained is more product focused, some more user behavior focused. I look at them as distinct. Design is a more general term that encompasses both of those.

File: guide/contribute/formatting.md (hunk 2 above)
Comment (GBKS) [reply guide/contribute/formatting.md:88]: Ah yes. We have another rake test error with an `href="#"` on that page. How about just changing the '#' to the https://bitcoin.design URL? That should get rid of the error and users should see the proper markup.

File: guide/resources/design-research.md (hunk 3 above)
Comment (GBKS) [reply guide/resources/design-research.md:3]: I'm down with it.

Comment (sbddesign) [reply guide/resources/design-research.md:3]: Agreed

**Review by sbddesign — CHANGES_REQUESTED** [review]
Looking good!
  
The `rake test` works for me after your recent updates. Fix the "Charles Eames" blockquote demo and then I will signal approval on this.

On a side note, how do we handle URI redirects with Netlify? Is there a config that we can add to the repo here, or must that be handled in Netlify admin?

Specifically, I would like to ensure that a URI like `/guide/designing-products/user-research/resources` is properly redirected to `/guide/resources/design-research/`. Happy to help with this if needed.

**Comment by sbddesign** [comment]
Once this is merged, perhaps we open up a header image for "Resources" page as a Good First Issue.

File: guide/contribute/formatting.md (hunk 2 above)
Comment (Bosch-0) [reply guide/contribute/formatting.md]: Fixed :)

**Comment by Bosch-0** [comment]
Fixed blockquote @sbddesign @GBKS  - everything else looks good to go!

File: guide/designing-products/wallet-interoperability.md [hunk 6]
```diff
@@ -232,11 +232,11 @@ Your application should support constructing and managing transaction files, als
 
 ---
 
-To kickstart your design work, take a look at [design resources]({{ '/guide/designing-products/design-resources/' | relative_url }}).
+Next, we cover how to onboard new users to a bitcoin daily spending wallet [onboarding]({{ '/guide/onboarding/' | relative_url }}).
```
Comment (sbddesign) [inline guide/designing-products/wallet-interoperability.md:235]:
```suggestion
Next, we cover how to onboard new users to a bitcoin daily spending wallet [onboarding]({{ '/guide/onboarding/introduction/' | relative_url }}).
```

Comment (sbddesign) [reply guide/designing-products/wallet-interoperability.md:235]: This link showed up broken in `rake test` and returns 404 on deploy preview, it should match the button below it.

File: guide/contribute/formatting.md [hunk 7]
```diff
@@ -85,7 +85,7 @@ There should be whitespace between paragraphs.
 ```markdown
 > The details are not the details. They make the design.
 >
-> <cite>By <a href="#">Charles Eames</a></cite>
+> <cite>By <a href="https://bitcoin.design ">Charles Eames</a></cite
```
Comment (sbddesign) [inline guide/contribute/formatting.md:88]:
```suggestion
> <cite>By <a href="https://bitcoin.design">Charles Eames</a></cite
```

Comment (sbddesign) [reply guide/contribute/formatting.md:88]: Removed stray space

**Review by sbddesign — CHANGES_REQUESTED** [review]
- The onboarding link you added on the Interoperability page is broken -- I left a suggestion to fix it
- Stray whitespace in the blockquote cite (left a suggestion)
- In `guide/contribute/formatting.md`, if you edit line 93 to match the link in line 88, it will remove the annoying error from `rake test` for good

File: guide/contribute/formatting.md [hunk 8]
```diff
@@ -85,7 +85,7 @@ There should be whitespace between paragraphs.
 ```markdown
 > The details are not the details. They make the design.
 >
-> <cite>By <a href="#">Charles Eames</a></cite>
+> <cite>By <a href="https://bitcoin.design">Charles Eames</a></cite
```
Comment (GBKS) [inline guide/contribute/formatting.md:88]:
```suggestion
> <cite>By <a href="https://bitcoin.design">Charles Eames</a></cite>
```

**Review by GBKS — APPROVED** [review]

**Review by sbddesign — APPROVED** [review]

**Pull request merged**